app/javascript/lib/mixpanel_event_tracking.js

```diff
@@ -1,5 +1,6 @@
 import "../vendor/navigator.sendbeacon.min.js";
-
+import Rails from "@rails/ujs";
+window.Rails = Rails;
```

Comment:
we can eliminate this dependency from this file (and potentially solve the issue w/ test scope you mention in the pr description) by lifting the csrf information from the DOM directly.